shared/protocol/src/test/java/io/pravega/shared/protocol/netty/AppendEncodeDecodeTest.java

```diff
@@ -407,20 +440,24 @@ public void testblockTimeout() throws Exception {
         Arrays.fill(content, (byte) 1);
         Event event = new Event(Unpooled.wrappedBuffer(content));
         idBatchSizeTrackerMap.remove(1L);
-        idBatchSizeTrackerMap.put(1L, new FixedBatchSizeTracker(3));
+        idBatchSizeTrackerMap.put(1L, new FixedBatchSizeTracker(appendBlockSize));
```

Comment:
where is `appendBlockSize` defined?

Reply:
its defined in the beginning of the 
public class AppendEncodeDecodeTest {

    private final int appendBlockSize = 1024;  
...
...
}